src/oss-config-preview/1.0.0/ControlSettings.json

```diff
@@ -883,6 +883,82 @@
       ]
     
     },
+    {
+      "ResourceType":"Build",
+      "ControlIds":[
+        "ADO_Build_AuthZ_Restrict_Broader_Group_Access",
+        "ADO_Build_SI_Dont_Use_Broadly_Editable_Task_Group",
+        "ADO_Build_SI_Dont_Use_Broadly_Editable_Variable_Group",
+        "ADO_Build_DP_Dont_Make_Secrets_Available_To_Forked_Builds",
+        "ADO_Build_SI_Review_URL_Variables_Settable_At_Queue_Time"
+      ]
+    
+    },
+    {
+      "ResourceType":"Release",
+      "ControlIds":[
+        "ADO_Release_AuthZ_Restrict_Broader_Group_Access",
+        "ADO_Release_SI_Dont_Use_Broadly_Editable_Task_Group",
+        "ADO_Release_SI_Dont_Use_Broadly_Editable_Variable_Group",
+        "ADO_Release_SI_Review_URL_Variables_Settable_At_Release_Time"
+    ]
+        
+    },
+    {
+      "ResourceType":"ServiceConnection",
+      "ControlIds":[
+        "ADO_ServiceConnection_AuthZ_Restrict_Broader_Group_Access",
+        "ADO_ServiceConnection_AuthZ_Dont_Grant_BuildSvcAcct_Permission",
+        "ADO_ServiceConnection_AuthZ_Dont_Grant_All_Pipelines_Access"
+    ]
+        
+    },
+    {
+      "ResourceType": "AgentPool",
+      "ControlIds": [
+        "ADO_AgentPool_AuthZ_Dont_Grant_All_Pipelines_Access",
+        "ADO_AgentPool_AuthZ_Dont_Enable_Auto_Provisioning",
+        "ADO_AgentPool_DP_Enable_Auto_Update",
+        "ADO_AgentPool_AuthZ_Restrict_Broader_Group_Access"
+      ]
+    },
+    {
+      "ResourceType": "VariableGroup",
+      "ControlIds": [
+        "ADO_VariableGroup_AuthZ_Dont_Grant_All_Pipelines_Access_On_VG_With_Secrets",
+        "ADO_VariableGroup_DP_No_PlainText_Secrets_In_Variables",
+        "ADO_VariableGroup_AuthZ_Restrict_Broader_Group_Access",
+        "ADO_VariableGroup_AuthZ_Restrict_Broader_Group_Access_On_VG_With_Secrets"
+      ]
+    },
+    {
+      "ResourceType": "Feed",
+      "ControlIds": [
+        "ADO_Feed_AuthZ_Restrict_Broader_Group_Access",
+        "ADO_Feed_AuthZ_Dont_Grant_BuildSvcAcct_Permission"
+      ]
+    },
+    {
+      "ResourceType": "SecureFile",
+      "ControlIds": [
+        "ADO_SecureFile_AuthZ_Dont_Grant_All_Pipelines_Access",
+        "ADO_SecureFile_AuthZ_Restrict_Broader_Group_Access"
+      ]
+    },
+    {
+      "ResourceType": "Environment",
+      "ControlIds": [
+        "ADO_Environment_AuthZ_Dont_Grant_All_Pipelines_Access",
+        "ADO_Environment_AuthZ_Restrict_Broader_Group_Access"
+      ]
+    },
+    {
+      "ResourceType": "Repository",
+      "ControlIds": [
+        "ADO_Repository_AuthZ_Dont_Grant_BuildSvcAcct_Permission",
+        "ADO_Repository_AuthZ_Dont_Grant_BuildSvc_Permission_On_Branch"
```

Comment:
pleas change prod config as well else you wil have to make another pr

Reply:
Made the changes